cli/config/config.go

```diff
@@ -140,6 +140,9 @@ func LoadDefaultConfigFile(stderr io.Writer) *configfile.ConfigFile {
 	if err != nil {
 		fmt.Fprintf(stderr, "WARNING: Error loading config file: %v\n", err)
 	}
+	if filepath.Base(configFile.Filename) == oldConfigfile {
```

Comment:
🤦 this doesn't work; I originally had it set the filename to `~/.dockercfg`, but when loading from a legacy file, I think we _write_ to the new file. Let me double check this